decidim-accountability/spec/permissions/decidim/accountability/admin/permissions_spec.rb

```diff
@@ -118,4 +118,20 @@
 
     it_behaves_like "permission is not set"
   end
+
+  describe "projects_import" do
+    let(:resource) { create :result, component: accountability_component }
+    let(:action_subject) { :projects_import }
+    let(:extra_context) { { projects_import: resource } }
```

Comment:
In the controller permission checks you are not providing the extra context variables, so they shouldn't be provided here either. Avoid writing unnecessary code you are not using anywhere.

Please also see the comments regarding the permission class itself before refactoring this test.